plugin.php

```diff
@@ -40,6 +40,7 @@ function classloader($class) {
 add_action('plugins_loaded', __NAMESPACE__ . '\Plugin::loadTextdomain');
 add_action('init', __NAMESPACE__ . '\Plugin::preInit', 0);
 add_action('init', __NAMESPACE__ . '\Plugin::init', 20);
+add_action('init', __NAMESPACE__ . '\ProductAttributePageTitle::init', 20);
```

Comment:
The contents of the new init function should have been added directly to Plugin::init() instead.  The point of our structure is to avoid loading unnecessary code on pages that do not need it.  By registering ProductAttributePageTitle::init() for the init hook here, the whole class is loaded unconditionally on all pages.